src/angular/ngWalker.ts

```diff
@@ -195,7 +195,7 @@ export class NgWalker extends Lint.RuleWalker {
     }
     const sf = ts.createSourceFile(path, `\`${content}\``, ts.ScriptTarget.ES5);
     const original = sf.getFullText;
-    sf.getFullText = function () {
+    sf.getFullText = function() {
```

Comment:
I have this rule set to `always` in my own projects but it would require space before parens also in methods. E.g.: `export function assertFailures<SPACE_REQUIRED>(ruleName: string, source: string, fails: IExpectedFailure[], options = null) {`

Specific reasoning is only to have a consistency. So either I will remove a rule or I will change it to `always` and apply the rule on all functions ...